### PR TITLE
7.0 sale stock sub state

### PR DESCRIPTION
**Pull request by chafique-delli**
This module allows to complete the sub state of the sale order
with the different stages of delivery.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Seeing the three modules, I see that there's no automation for assigning this sub state depending on the other flows, so I vote for:
- Join the three in one module.
- Don't depend in other modules but on sale.

**Comment by bealdav** [comment]
Maybe it would useful to change this sub state according to the number of shipped products:
i.e. : sent', or 'partially shipped'

**Comment by eLBati** [comment]
@pedrobaeza this module overrides `stock.picking` and `stock.move`. It could not depend on sale only

File: sale_stock_sub_state/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{
+    'name': 'Sale Stock Sub State',
+    'version': '0.1',
+    'category': 'Sales Management',
+    'license': 'AGPL-3',
+    'description': """
+    This module allows to complete the sub state of the sale order
+    with the different stages of delivery.
+    """,
+    'author': 'Akretion',
```
Comment (gurneyalex) [inline sale_stock_sub_state/__openerp__.py:31]: please add Odoo Community Association (OCA) tot the autors list.

**Comment by gurneyalex** [comment]
could you please add anautomated test?

**Comment by pedrobaeza** [comment]
Closing as this seems abandoned without repairing comments. If not, please feel free to reopen.

**Pull request closed without merging**